### ISSUE TITLE
Adding dataset from ECDC

### ISSUE THREAD

**Issue by gatemezing** [gengchenmai/covid-19-kg#30]
Hi,
I've seen this great work on LOD public list and want to contribute.
I've seen this dataset from ECDC (European Centre for Disease Prevention and Control) https://data.europa.eu/euodp/en/data/dataset/covid-19-coronavirus-data which might be of interest. Of course, they also report cases and deaths, but it seems that the values can be interesting to compare with other sources. 
I've converted into RDF using popular vocabs using `qb:Observation`. 
Below is a sample Turtle file for Afghanistan on March 27th, 2020. 

```
ecdcr:af270320 a qb:Observation;
    qb:dataSet  ecdc-dataset: ;
    sdmx-dimension:refTime "2020-03-27"^^xsd:date;
    time:day "27"^^xsd:gDay;
    time:month "03"^^xsd:gMonth;
    time:year  "2020"^^xsd:year;
    covid19:deaths "0"^^xsd:integer ;
    covid19:confirmed "0"^^xsd:integer ;
    sdmx-dimension:refArea "AF" ;
    dct:coverage ecdcl:af .

    ecdcl:af a dct:Location ; 
    	rdfs:label "Afghanistan" ; 
    	geop:populationYear "2008"^^xsd:gYear ; 
    	idemo:population "37172386" ;

    .
```
I put the current RDF file [here](http://linkedvocabs.org/dataset/covid19/covid-ecdc.ttl) and let me know if it could help. Of course, the URIs of the resources can be changed since they are not deref. HTH

**Comment by kjano** [comment]
Thanks. @gatemezing do you mean to compare this to the JHU data we use? Could you use SOSA/SSN for the observations

**Comment by gatemezing** [comment]
Yes, @kjano. Since this dataset offers a day by day observation, not aggregation/cumulative data. Sure, I can use SOSA/SSN for the observations if it fits your needs. BTW, will it be useful to use SOSA/SSN here since we don't have "sensors" here? The numbers come from multiple sources, such as websites of ministries and public health institutes (see https://qap.ecdc.europa.eu/public/extensions/COVID-19/COVID-19.html). IMHO, aggregating different "views" can be useful, as now some people are saying that the cases reported by different countries are not "comparable".
